source/brailleDisplayDrivers/brailliantB.py

```diff
@@ -34,7 +38,8 @@
 HR_POWEROFF = "\x07"
 
 KEY_NAMES = {
-	# Braille keyboard.
+	1: "power", # Brailliant BI 32, 40 and 80.

```

Comment:
When will this key be received by NVDA?

Reply:
The documentation does not say exactly, but clearly suggests it could be sent. The reason I say this is because there is a specific exception in the documentation for BI 14 and BrailleNote touch which says that their power keys are not sent to the host.

Reply:
> When will this key be received by NVDA?

When you press the power button on the Brailliant. It doesn't do anything useful, but it's better than having an unrecognised key.